README.md

```diff
@@ -1,2 +1,118 @@
-# strigo-github-exercises-examples
-Examples for markdown exercises that may be used in Strigo training events
+# Strigo GitHub Class Exercises examples
+
+## Quick start
+
+1. Create `.strigo/config.yml` file in your exercises' repository
+1. Configure exercises in `.strigo/config`
+
+   ```yaml
+   ---
+   classes:
+     - exercises:
+         # `file` is relative to .strigo directory
+         - file: "../Exercise.md"
+           title: "Title!"
+   ```
+
+1. In Strigo Class editor, fetch Exercises for a desired Class
+1. Done!
+
+---
+
+## Config file structure
+
+- `classes` - array of Classes
+  - `class_id` - defines the Class ID, optional if the configuration has only one Class
+  - `exercises` - array of Class' Exercises, cannot be empty
+    - `file` - path to Exercise markup file, relative to `.strigo` directory, supported extensions `.md` (markdown) and `.rst` (restructured), any other extension should be overriden by `syntax` field (see below)
+    - `title` - defines Exercise title in Strigo interface
+    - `syntax` - overrides `file` extension, supported syntaxes `.md` (markdown) and `.rst` (restructured)
+
+---
+
+## Main concepts
+
+1. **Hello world**
+
+   Just configure the order of the files, set titles and it is good to go.
+
+   ```yaml
+   ---
+   classes:
+     - exercises:
+         # `file` is relative to .strigo directory
+         - file: "../hello-world/hello.md"
+           title: "Hello!"
+         - file: "../hello-world/world.txt"
+           title: "World!"
+   ```
+
+1. **Multiple Classes in a single repository**
+
+   You can have multiple Classes in a single repository, defining Class IDs for each.
```

Comment:
have => support
Classes => classes
defining Class IDs for each => by defining class IDs for each class.

Reply:
"support"... not sure it means what we try to mean

Reply:
Hmm. "configure"?